base_layer/core/src/blocks/block_header.rs

```diff
@@ -364,18 +368,67 @@ pub(crate) mod hash_serializer {
     }
 }
 
+pub(crate) mod fixed_hash_serializer {
```

Comment:
Strange place for this. Shouldn't this live with FixedHash ?

Reply:
This is in the same place as `pub(crate) mod hash_serializer `

When trying to move:
_Module `tari_core::blocks::block_header::fixed_hash_serializer` uses module `tari_core::blocks::block_header` which will be inaccessible after move_